source/common/secret/sds_api.cc

```diff
@@ -17,27 +17,25 @@ SdsApi::SdsApi(const LocalInfo::LocalInfo& local_info, Event::Dispatcher& dispat
                const envoy::api::v2::core::ConfigSource& sds_config,
                const std::string& sds_config_name, std::function<void()> destructor_cb,
                Api::Api& api)
-    : local_info_(local_info), dispatcher_(dispatcher), random_(random), stats_(stats),
+    : init_target_(
+          fmt::format("SdsApi {}", sds_config_name),
+          [this]() {
```

Comment:
Prefer maybe to make this a private method or anonymous namespace and reference it rather than inline; otherwise this C++ starts to look like Javascript..